staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -176,6 +177,15 @@ func WithPriorityAndFairness(
 			}()
 
 			execute := func() {
+				startedAt := time.Now()
+				defer func() {
+					// TODO: Once PR#104557 merges, we can remove the 'klog.V(3).Enabled()' check.
```

Comment:
Hmm - I'm not sure it will ever merge, because of : https://github.com/kubernetes/kubernetes/pull/104557#issuecomment-904944671

Reply:
now, it's not static - the filter is always enabled and i am checking runtime for each request if log level 3 is enabled, only then the httplog construct is initialized. 

once https://github.com/kubernetes/kubernetes/pull/104920 merges I will ask David to remove the hold.

Reply:
Yes, I think leaving the `V(3).Enabled()` check here permanently is fine.